manifests/b/Badlion/BadlionClient/3.15.1/Badlion.BadlionClient.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Badlion.BadlionClient
+PackageVersion: 3.15.1
+PackageLocale: en-US
+Publisher: ELS Gaming Online
+PackageName: Badlion Client
+License: Proprietary
+Copyright: Copyright © 2023 Badlion
+ShortDescription: Badlion Client

```

Comment:
```suggestion

PackageLocale: en-US
Publisher: Badlion
# PublisherUrl:
PublisherSupportUrl: https://support.badlion.net/
# PrivacyUrl:
# Author:
PackageName: Badlion Client
# PackageUrl:
License: Proprietary
# LicenseUrl:
Copyright: © 2013-2023 ESL Gaming Online, Inc.
# CopyrightUrl:
ShortDescription: Badlion Client Minecraft Launcher
Description: Badlion Client is a Minecraft launcher with many added features. It includes anticheat, built-in FPS boost and other mods, and is compatible with most versions of minecraft 1.7 or newer.
Moniker: blc
Tags:
- minecraft
- optifine
# Agreements:
# ReleaseNotes:
# ReleaseNotesUrl:
# PurchaseUrl:
# InstallationNotes:
Documentations:
- DocumentLabel: FAQ
  DocumentUrl: https://client.badlion.net/faq
```
Publisher must match what is in control panel. Additional metadata should match existing Badlion client metadata